### PR TITLE
CI-246: Shoutout Button - ability open shoutout modal window on each page

### PR DESCRIPTION
**Pull request by VitaliiShevchenko**
[CI-246](https://cloverpop-internship.atlassian.net/browse/CI-246)

[CI-246]: https://cloverpop-internship.atlassian.net/browse/CI-246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

Together with ability to open shoutout button on each page was take off all HUD elements from each page and put to separately  CornerElements and according was fixed every pages which breaking because HUD elements was to inleted content of pages and so all HUD elements stopped  to jump when user going from page to page

### CODE REVIEW

**Review by Maryna-Harasko — CHANGES_REQUESTED** [review]
@VitaliiShevchenko, please return the <Header> and <Menu> components. In the epic CI-305 "Drop-down" that I'm currently working on, everything is tied to these components, but the Shoutout button is not connected to it.

**Review by Maryna-Harasko — APPROVED** [review]

File: app/javascript/components/Pages/EmotionEntry.js [hunk 1]
```diff
@@ -125,13 +126,15 @@ const EmotionEntry = ({data, setData, saveDataToDb, steps, service}) => {
             <Emojis show={show}/>
           </CSSTransition>
         </div>
-        <div className='d-flex justify-content-between m-3'>
-          <ShoutOutIcon/>
+        <div className='d-flex position-absolute placement-buttons justify-content-between col-6 offset-3'>
```
Comment (SokalMet) [inline app/javascript/components/Pages/EmotionEntry.js:129]: are you sure **col-6 offset-3** is useful here? I do not see the **row** class above. Maybe **w-50** could work better here?

File: app/javascript/components/Pages/IcebreakerAnswer.js [hunk 2]
```diff
@@ -96,15 +97,17 @@ const IcebreakerAnswer = ({data, setData, saveDataToDb, steps, service}) => {
                 </div>
               </div>
             </div>
-            <div className='d-flex justify-content-between m-3'>
-              <ShoutOutIcon/>
-              <BtnBack addClass="answer-custom" onClick={backHandling}/>
-              <BtnPrimary addClass="answer-custom" onClick={handlingOnClickNext} text={isEmptyStr(answerBody) ? 'Skip to Results' : 'Submit'} />
-              <HelpIcon/>
+            <div className='d-flex placement-buttons justify-content-between col-6 offset-3 pb-53 mt-5'>
```
Comment (SokalMet) [inline app/javascript/components/Pages/IcebreakerAnswer.js:100]: the same - **col-6 offset-3**. maybe w-50 is better here?

File: app/javascript/components/Pages/IcebreakerQuestion.js [hunk 3]
```diff
@@ -91,15 +90,17 @@ const IcebreakerQuestion = ({data, setData, saveDataToDb, steps, service}) => {
                 }
               </div>
             </div>
-            <div className='d-flex justify-content-between m-3'>
-              <ShoutOutIcon/>
-              <BtnBack addClass="answer-custom" onClick={backHandling}/>
-              <BtnPrimary addClass="answer-custom" onClick={handlingOnClickNext} text={isEmptyStr(funQuestionBody) ? 'Skip to Results' : 'Submit'} />
-              <HelpIcon/>
+            <div className='d-flex placement-buttons justify-content-between col-6 offset-3 pb-53 mt-5'>
```
Comment (SokalMet) [inline app/javascript/components/Pages/IcebreakerQuestion.js:93]: the same

**Review by SokalMet — APPROVED** [review]

File: app/javascript/components/Pages/EmotionEntry.js (hunk 1 above)
Comment (VitaliiShevchenko) [reply app/javascript/components/Pages/EmotionEntry.js:129]: Yes of course this is work in this case very well and I think this decision more preferable in this case.

File: app/javascript/components/Pages/IcebreakerAnswer.js (hunk 2 above)
Comment (VitaliiShevchenko) [reply app/javascript/components/Pages/IcebreakerAnswer.js:100]: I think this decision more preferable in this case.

File: app/javascript/components/Pages/IcebreakerQuestion.js (hunk 3 above)
Comment (VitaliiShevchenko) [reply app/javascript/components/Pages/IcebreakerQuestion.js:93]: Thank you  I think this decision more preferable in this case.

**Pull request merged**